Report_templates/depthprofiles.Rmd

```diff
@@ -191,19 +192,26 @@ if(length(dts) > 0){
 
   #add titles around the plot
   title(ylab = "DEPTH, IN METERS",
-        outer = TRUE, line = ylabel_line)
-  title(xlab = specifcond_title,
-        outer = TRUE, line = 2)
-  title(xlab = PH_title,
-        outer = TRUE, line = -24)
-  title(xlab = h2otemp_title,
-        outer = TRUE, line = -27.5)
-  title(xlab = DO_title,
-        outer = TRUE, line = -47.8)
-  title(xlab = plot_sub_title, 
-          outer = TRUE, line = -49)
+          outer = TRUE, 
+          line = -1)
```

Comment:
I think if you change this back to `ylabel_line` that should fix the issue you just mentioned in https://github.com/USGS-R/lakeReport/issues/74#issuecomment-396341661 ?